client/my-sites/site-settings/custom-content-types/style.scss

```diff
@@ -13,4 +13,11 @@
 .custom-content-types__label,
 .custom-content-types__module-settings .form-toggle__label-content {
 	font-weight: 600;
-}
\ No newline at end of file
+}
+
+html:not( [lang='de'] ) .custom-content-types__indented-form-field {
```

Comment:
Let's be careful here because this is likely to break other languages too, and according to the code, I'm guessing we are acknowledging that German will be broken here, but what about Swiss-German? What about Allemanisch?

I'd ask more deeply what is the purpose of lower-casing here? what goal are we trying to solve? we can probably solve it in a different way that doesn't rely on us knowing everything about every language :wink:

Reply:
> I'd ask more deeply what is the purpose of lower-casing here?

Really just to make the concatenated sentence look less horrible, until we decide if it's worth adding new strings here. I'm excluding German as it's a special case, and one of our popular languages. I tested with our top 16 languages and there was no visible detriment. 

But that doesn't solve the main problem...

> we can probably solve it in a different way that doesn't rely on us knowing everything about every language

Reply:
Often I'm able to work around concatenation by separating the passed variable from the rest of the sentence. It's maybe not the best solution but I think it can go pretty far.

```
number to show per site: posts
number to show per site: pages
…
```

it's ugly and needs some editorial/design review but now I can embed the string in the translation…

```js
translate( 'number to show per site: %(typeOfThingToShow)s', … )
```

carry on…